src/views/SystemProfile/components/SystemData/index.tsx

```diff
@@ -0,0 +1,372 @@
+                      term={t('singleSystem.systemData.apiGateway')}
+                    />
+                    <i className="fa fa-exclamation-circle text-base-light display-inline-flex" />
+                  </div>
+                  <DescriptionDefinition
+                    className="line-height-body-3 margin-bottom-2"
+                    definition="Kong"
+                  />
+                  <UswdsReactLink to="tools-and-software">
+                    {t('singleSystem.systemData.viewGateway')}
+                  </UswdsReactLink>
+                </Grid>
+                <Grid tablet={{ col: 6 }} className="margin-bottom-5">
+                  <div>
+                    <DescriptionTerm
+                      className="display-inline-flex margin-right-1"
+                      term={t('singleSystem.systemData.access')}
+                    />
+                    <i className="fa fa-exclamation-circle text-base-light display-inline-flex" />
+                  </div>
+                  <DescriptionDefinition
+                    className="line-height-body-3"
+                    definition="Internal and external access"
+                  />
+                </Grid>
+              </Grid>
+
+              {/* TODO: Map and populate tags with CEDAR */}
+              <h3 className="margin-top-0 margin-bottom-1">
+                {t('singleSystem.systemData.dataCategories')}
+              </h3>
+              {system?.developmentTags?.map((tag: string) => (
+                <Tag
+                  key={tag}
+                  className="system-profile__tag text-base-darker bg-base-lighter margin-bottom-1"
+                >
+                  {tag} {/* TODO: Map defined CEDAR variable once availabe */}
+                </Tag>
+              ))}
+            </SectionWrapper>
+
+            <SectionWrapper
+              borderBottom={isMobile}
+              className="margin-bottom-4 padding-bottom-5"
+            >
+              <h2 className="margin-top-0">
+                {t('singleSystem.systemData.dataExchanges')}
+              </h2>
+
+              <CardGroup className="margin-0">
+                {system?.systemData?.map((data: tempSystemDataProp) => {
+                  return (
+                    <Card key={data.id} className="grid-col-12 margin-bottom-2">
+                      <CardHeader className="padding-2 padding-bottom-0">
+                        <div className="margin-bottom-0 easi-header__basic flex-align-baseline">
+                          <h3 className="margin-top-0 margin-bottom-1">
+                            {data.title}
+                          </h3>
+                          <Tag
+                            className={classnames(
+                              'font-body-md',
+                              'margin-bottom-1',
+                              {
+                                'bg-success-dark text-white':
+                                  data.status === 'Active' ||
```

Comment:
I'm not a huge fan of having the status be dependent on string equality. No need to change this now, since it's temporary data, but let's revisit what this check looks like once we know what the GraphQL type looks like.

Reply:
Even if we end up having to take a string from the GraphQL data and map it to success/warning/not-applicable, I think it'd be better to have a standalone function that encapsulates that logic, and looks something like:
```ts
const identifySystemDataStatus(status: string): 'success' | 'warning' | 'notApplicable' => {
  // map status strings to success/warning/not-applicable
}
```

Reply:
Yep makes sense, I agree.  I'll leave for now, and once we get CEDAR, we can begin mapping these appropriately.